### ISSUE TITLE
Issue: [9.0.0] Dashboard does not start. 'frontmatter.dashboard' not found error.

### ISSUE THREAD

**Issue by Ocanamat** [estruyf/vscode-front-matter#641]
**Describe the bug**

- Opening the FrontMatter extension results in an endless loading sidebar display.
- Launching the dashboard from the command palette shows the 'frontmatter.dashboard' not found error.

**Screenshots**
![FrontMatter_not_Loading](https://i.imgur.com/w5KeIvr.gif)

**Desktop (please complete the following information):**
VS Code Version Info

> Version: 1.81.1 (system setup)
> Commit: 6c3e3dba23e8fadc360aed75ce363ba185c49794
> Date: 2023-08-09T22:22:42.175Z
> Electron: 22.3.18
> ElectronBuildId: 22689846
> Chromium: 108.0.5359.215
> Node.js: 16.17.1
> V8: 10.8.168.25-electron.0
> OS: Windows_NT x64 10.0.19045

**Additional context**

- The project is a Hugo alchemy-wow website.
- Project contains a `frontmatter.json` and a `.frontmatter` folder.

**Temporary Solution**

- Downgraded the extension to v8.4.0 via the extension page within VSCode and the functionality returned to normal. 
![image](https://github.com/estruyf/vscode-front-matter/assets/13169121/17e0a034-d227-4e8c-909f-1af153ba3f25)

**Comment by estruyf** [comment]
I have tried to reproduce it in several VSCode instances (Windows and macOS) but haven't been able to reproduce it. 

Still trying to figure out where VSCode fails to load the panel, as the loader is something from VSCode. 

Can you try reinstalling the latest version again and check if you see something in the VSCode output (`ctrl+shift+u`)? Verify the `extension host` and `front matter cms` outcomes.

You can also try the beta version to see if the same happens.

**Comment by Ocanamat** [comment]
Hi Elio,

Thanks for the suggestion: I already tried reinstalling to no results.

Looking into the `extension host` output of the terminal, I got the following:
``` 
2023-08-30 14:02:09.912 [error] Activating extension eliostruyf.vscode-front-matter failed due to an error:
2023-08-30 14:02:09.912 [error] Error: ENOENT: no such file or directory, open '\Users\[USER]\.vscode\extensions\eliostruyf.vscode-front-matter-9.0.0\l10n\bundle.l10n.json'
	at Object.openSync (node:fs:596:3)
	at Object.func [as openSync] (node:electron/js2c/asar_bundle:2:1822)
	at readFileSync (node:fs:464:35)
	at t.readFileSync (node:electron/js2c/asar_bundle:2:9165)
	at Object.b (f:\Users\Tamanaco\.vscode\extensions\eliostruyf.vscode-front-matter-9.0.0\dist\vendors.859af0e97ef15122e826.js:2:139674)
	at f:\Users\Tamanaco\.vscode\extensions\eliostruyf.vscode-front-matter-9.0.0\dist\extension.js:1:87452
	at Generator.next (<anonymous>)
	at f:\Users\Tamanaco\.vscode\extensions\eliostruyf.vscode-front-matter-9.0.0\dist\extension.js:1:86939
	at new Promise (<anonymous>)
	at n (f:\Users\Tamanaco\.vscode\extensions\eliostruyf.vscode-front-matter-9.0.0\dist\extension.js:1:86684)
	at t.activate (f:\Users\Tamanaco\.vscode\extensions\eliostruyf.vscode-front-matter-9.0.0\dist\extension.js:1:87247)
	at Dn.eb (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:116:13102)
	at Dn.db (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:116:12816)
	at c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:116:10913
	at async p.n (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:102:6206)
	at async p.m (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:102:6169)
	at async p.l (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:102:5626)
```
And the `front matter cms` output option is missing from the slide down menu.

Thanks for the help!

**Comment by estruyf** [comment]
Interesting, can you verify what is added in this path: `\Users\[USER]\.vscode\extensions\eliostruyf.vscode-front-matter-9.0.0\l10n\`

**Comment by estruyf** [comment]
I updated the VSCode file parsing method. That might make a difference on your machine. Can you verify this in beta version: `9.1.6025791`.

You can find the beta version here: https://marketplace.visualstudio.com/items?itemName=eliostruyf.vscode-front-matter-beta 

**Important**: uninstall/disable the main version, as only one can be running.

**Comment by estruyf** [comment]
Just released 9.1.0 which includes this fix.

**Comment by Ocanamat** [comment]
Thanks! 🎉
